.github/workflows/ubuntu_20.yml

```diff
@@ -132,7 +132,8 @@ jobs:
       runner: "[ 'self-hosted', 'igpu' ]"
       container: '{"image": "ubuntu:20.04", "volumes": ["/dev/dri:/dev/dri"], "options": "--group-add 109 --group-add 44
         --device /dev/dri:/dev/dri"}'
-    if: fromJSON(needs.smart_ci.outputs.affected_components).GPU
+    # if: fromJSON(needs.smart_ci.outputs.affected_components).GPU
+    if: ${{ 'false' }} # Ticket: 152280
```

Comment:
Don't we need to exclude it also from overall status job?

Reply:
It is needed otherwise the overall status job would not start and complete: https://github.com/openvinotoolkit/openvino/pull/26559/files#diff-574d408a6c27aae6aec912e597bedfbb2bbe7ace536e85fdb50624133975138bR158

Reply:
Fixed